apps/web/src/app/(main)/onboarding/form/create-form-dialog.tsx

```diff
@@ -0,0 +1,169 @@
+'use client';
+
+import React, { useState, useTransition } from 'react';
+import { useRouter } from 'next/navigation';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { InfoCircledIcon } from '@radix-ui/react-icons';
+import { useForm } from 'react-hook-form';
+import { toast } from 'sonner';
+import { z } from 'zod';
+
+import { Button } from '@formbase/ui/primitives/button';
+import {
+  Dialog,
+  DialogContent,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from '@formbase/ui/primitives/dialog';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@formbase/ui/primitives/form';
+import { Input } from '@formbase/ui/primitives/input';
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from '@formbase/ui/primitives/tooltip';
+
+import { api } from '~/lib/trpc/react';
+
+const FormSchema = z.object({
+  name: z.string().min(1, {
+    message: 'Form title is required.',
+  }),
+  returnUrl: z.string().optional(),
+  description: z.string().optional(),
+  keys: z.array(z.string()).optional(),
+});
+
+export function CreateFormDialog() {
+  const router = useRouter();
+  const form = useForm<z.infer<typeof FormSchema>>({
+    resolver: zodResolver(FormSchema),
+    defaultValues: {
+      name: '',
+      description: '',
+      returnUrl: '',
+      keys: [''],
+    },
```

Comment:
Since this is the create form dialog in the onboarding process, we can do away with it. I don't think we will substitute the default empty things with something else (updating)

Reply:
i dont get this

Reply:
the default values